### PR TITLE
feat: integrate gen_svg into joyrun_sync.py(#675)

### LINKED ISSUE #675: [Function enhancement] SVG info source error caused by the inability to automate JoyRun
## Background：

After 
```
python3(python) run_page/joyrun_sync.py ${your mobile} ${your 验证码}
pip3 install -r requirements.txt
npm install -g corepack && corepack enable && pnpm install
pnpm develop
``` 
running activities data can be seen in the web. But when clicking `Total` in years, the GithubSvg and GridSvg will show yihong0618's info but not mine.

### option 1[I prefer and what I did manually to resolve it]:

Update `joyrun_sync.py` to support `Make svg GitHub profile` action in Run Data Sync. 
After sync data from joyRun, the SVG will also be generated.

Cons:
- Need more parameters when running `joyrun_sync.py` 
- Need further discussion about how to integrate these script to one
  - By command invoking a new process?
  - By Module import?

### option 2[I think fine]:
 
Update readme to indicate it.

### PR DESCRIPTION
**Pull request by wayichao**
To resolve #675 

By `subprocess.run` to serial start process.

Tried to investigate integrate `gen_syg` as module, but `sys.argv` must be updated before invoking `gen_svg.main()`. Repeatedly constructing `sys.argv` 3 times is not elegant to me.

Verified locally
- with 2 new options
- without 2 new options
- with only `--athlete`
- with only `--min_grid_distance` // same behavior with without 2 new options is as expected

Ran `black .` but only kept changes by this commit.

### CODE REVIEW

**Review by yihong0618 — COMMENTED** [review]
nice job will take a look this weekend

File: run_page/joyrun_sync.py [hunk 1]
```diff
@@ -336,12 +337,95 @@ def get_all_joyrun_tracks(self, old_tracks_ids, with_gpx=False):
...
+            "--special-color2",
+            "red",
+            "--output",
+            "assets/grid.svg",
+            "--use-localtime",
+            "--min-distance",
+            str(min_grid_distance),
+        ],
+        [
+            "python",
```
Comment (yihong0618) [inline run_page/joyrun_sync.py:399]: this maybe python3

can we sue sys.executable?

Comment (twyc) [reply run_page/joyrun_sync.py]: Updated. 
Learned it new to me~
https://docs.python.org/3/library/sys.html#sys.executable

**Comment by yihong0618** [comment]
thanks

**Pull request merged**